src/SmartComponents/modules/appReducer.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers } from 'redux';
+import { compareReducer } from './reducers';
+import { addSystemModalReducer } from '../DriftPage/AddSystemButton/addSystemModalReducer';
+import { errorAlertReducer } from '../ErrorAlert/errorAlertReducer';
+import { filterDropdownReducer } from '../DriftPage/FilterDropDown/filterDropdownReducer';
+import { actionKebabReducer } from '../DriftPage/ActionKebab/actionKebabReducer';
+
+export const appReducer = combineReducers({
+    compareState: compareReducer,
```

Comment:
Why does the name of the reducer change here from `compareReducer` to `compareState` ?

Reply:
So, I think this will answer this question and the one above:

![image](https://user-images.githubusercontent.com/15373136/59123519-673a8c00-892b-11e9-9c4a-3f2b13b1bb25.png)

Yes, all slices live within a single state known as appState (GO MOUNTAINNEERS!).
I renamed compareReducer to compareState to better represent what the object was. The compareReducer is the function above, while the object compareReducer returns is the compareState.

I would ultimately like to get rid of the compareState and appState altogether so everything lives within state.<whatever>.